common/lib/xmodule/xmodule/js/fixtures/matlabinput_problem.html

```diff
@@ -1,43 +1,51 @@
+  <div>
+    <span>
+      <p>
+      <p></p>
+    </span>
+    <span>
+      <section id="textbox_test_matlab_plot1_2_1" class="capa_inputtype cminput">
+        <textarea rows="10" cols="80" name="input_i4x-MITx-2_01x-problem-test_matlab_plot1_2_1"
+                  aria-describedby="answer_i4x-MITx-2_01x-problem-test_matlab_plot1_2_1"
+                  id="input_i4x-MITx-2_01x-problem-test_matlab_plot1_2_1" data-tabsize="4" data-mode="octave"
+                  data-linenums="true" style="display: none;">This is the MATLAB input, whatever that may be.
+        </textarea>
 
-  <div class="grader-status" tabindex="-1">
-      <span id="status_test_matlab_plot1_2_1" class="processing" aria-describedby="input_test_matlab_plot1_2_1">
-          <span class="status sr">processing</span>
-      </span>
-      <span style="display:none;" class="xqueue" id="test_matlab_plot1_2_1">1</span>
+        <div class="grader-status" tabindex="-1">
+          <span id="status_test_matlab_plot1_2_1" class="processing" aria-describedby="input_test_matlab_plot1_2_1">
+              <span class="status sr">processing</span>
+          </span>
+          <span style="display:none;" class="xqueue" id="test_matlab_plot1_2_1">1</span>
+          <p class="debug">processing</p>
+        </div>
+        <span id="answer_test_matlab_plot1_2_1"></span>
 
+        <div class="external-grader-message" aria-live="polite">
+          Submitted. As soon as a response is returned, this message will be replaced by that feedback.
+        </div>
+        <div class="ungraded-matlab-result" aria-live="polite">
+        </div>
 
-    <p class="debug">processing</p>
+        <div class="plot-button">
+          <input type="button" class="save" name="plot-button" id="plot_test_matlab_plot1_2_1" value="Run Code">
+        </div>
+      </section>
+    </span>
   </div>
-
-  <span id="answer_test_matlab_plot1_2_1"></span>
-
-  <div class="external-grader-message" aria-live="polite">
-    Submitted. As soon as a response is returned, this message will be replaced by that feedback.
-  </div>
-  <div class="ungraded-matlab-result" aria-live="polite">
-    
-  </div>
-
-  <div class="plot-button">
-      <input type="button" class="save" name="plot-button" id="plot_test_matlab_plot1_2_1" value="Run Code">
-  </div>
-
-  
-</section></span>
-</div>
-    </div>
   <div class="action">
     <input type="hidden" name="problem_id" value="Plot a straight line">
     <button class="reset" data-value="Reset">Reset<span class="sr"> your answer</span></button>
     <button class="show"><span class="show-label">Show Answer</span> </button>
   </div>
+
+  <div class="notification warning notification-gentle-alert is-hidden" tabindex="-1">
```

Comment:
Note to reviewers: This is the only real change in this file. I reorganized it to be cleaner and more organized for readability and format.